strawberryfields/apps/graph/resize.py

```diff
@@ -331,7 +330,8 @@ def clique_swap(clique: list, graph: nx.Graph, node_select: str = "uniform") ->
 
     if c_1:
         if node_select == "uniform":
-            swap_nodes = random.choice(c_1)
+            swap_index = np.random.choice(len(c_1))
```

Comment:
There was some usages of `random` from the Python standard library and other usages of `np.random`, but we should be consistent to prevent confusion. I decided to go with `np.random` - but in this case `np.random.choice` expects something that looks like a 1D array, while `c_1` is a list of tuples and hence looks like a 2D array such that `np.random.choice` would raise an exception if `c_1` were fed in.

This change allows the use of `np.random.choice` with just an int as input, and randomly picks one of the indices of `c_1`.

Reply:
For the extra curious, I did some poking around to see if there were any differences and came across this neat explanation: https://stackoverflow.com/a/7030595/281530

Reply:
Nice! Thanks @jswinarton